reference/admin.yaml

```diff
@@ -6539,6 +6539,163 @@ components:
           description: >
             Determines whether this organization has been verified by a
             verifier.
+    statements_in:
```

Comment:
Summary : 
Have added not null constraint for -> also have added in doc and script
file_name
file_size
no_of_debit
no_of_credit
no_of_transactions